src/app/components/navbar/navbar.component.ts

```diff
@@ -1,15 +1,23 @@
 import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute, NavigationEnd } from '@angular/router';
 
 @Component({
   selector: 'app-navbar',
   templateUrl: './navbar.component.html',
   styleUrls: ['./navbar.component.scss']
 })
 export class NavbarComponent implements OnInit {
+  currentRoute = '';
 
-  constructor() { }
+  constructor(private route: ActivatedRoute, private router: Router) { }
```

Comment:
Remove ActivatedRoute dependency since it is not being used

Reply:
Done